### PR TITLE
Edits from the community tools meeting (change font size and make it all single page for Helm)

### PR DESCRIPTION
**Pull request by JakeMakesStuff**
## Type of Change
- **Tool Source:** Vue source

## What issue does this relate to?

Fixes: #11 

### What should this PR do?

- change font size
- Make it all single page for Helm
- Add back button
- Add reference links
- Add explainer on homepage
- Add basic left rail navigation (although needs extra design)
- accordian style listing
- Collapsable with collapse all button

### What are the acceptance criteria?

Do you like how it is handled?

### CODE REVIEW

**Comment by MattIPv4** [comment]
Lgtm but we should wait for a design from @amnotadev before continuing with building the single page layout.

**Comment by JakeMakesStuff** [comment]
Note for Matt: As we discussed, the do-vue buttons seem to have a styling bug

File: src/kubernetes-tool/templates/app.vue [hunk 1]
```diff
@@ -20,25 +20,41 @@ limitations under the License.
...
+                </div>
+                <div class="column">
+                    <Header :title="i18n.templates.app.title">
+                        <template v-slot:description>
+                        </template>
+                        <template v-slot:header>
+                        </template>
+                        <template v-slot:buttons>
+                            <a class="button" @click="mainMenu">{{ i18n.templates.shared.mainMenu }}</a>
+                            <a class="button" @click="collapseHook">{{ i18n.templates.app.collapseAll }}</a>
```
Comment (MattIPv4) [inline src/kubernetes-tool/templates/app.vue]: This button currently just toggles the state of each "collapisble" - this should store it's own state and actually collpase/uncollapse everything. The text can then also reflect this.

File: src/kubernetes-tool/descriptions/README.md [hunk 2]
```diff
@@ -29,4 +29,4 @@ export default { ..., Filename }
 
 ## Kind Descriptions
 
-Descriptions of the kinds of deployments belong in `text_descriptions.ts`. These are used during categorisation.
+Descriptions of the kinds of deployments belong in `text_descriptions.ts`. These are used during categorisation. Additionally, the "Learn more..." lnks can be found in `learn_more.ts`.
```
Comment (MattIPv4) [inline src/kubernetes-tool/descriptions/README.md]:
```suggestion
Descriptions of the kinds of deployments belong in `text_descriptions.ts`. These are used during categorisation. Additionally, the "Learn more..." links can be found in `learn_more.ts`.
```

File: src/kubernetes-tool/templates/app.vue [hunk 3]
```diff
@@ -20,25 +20,41 @@ limitations under the License.
...
-                <template v-slot:description>
-                </template>
-                <template v-slot:header>
-                </template>
-                <template v-slot:buttons>
-                    <a class="button" @click="mainMenu">{{ i18n.templates.shared.mainMenu }}</a>
-                </template>
-            </Header>
+            <div class="columns">
+                <div class="column is-2" style="height: 100%; top: 0; position: sticky;">
```
Comment (MattIPv4) [inline src/kubernetes-tool/templates/app.vue]: This does not work on small screens at all. Need to make this collapsible. If there isn't space for it to fit side-by-side, it should float on top and be collapsed by default.

1280x800:
image

File: src/kubernetes-tool/templates/categorisation_view.vue [hunk 4]
```diff
@@ -1,41 +1,99 @@
...
-                <a v-for="item in all.get(key)" :key="item.fp" class="button is-small" :style="{marginRight: '10px', marginBottom: '10px'}" @click="emitFp(item.fp)">
-                    {{ item.fp }}
-                </a>
-            </p>
-            <hr v-if="index !== Array.from(all.keys()).length - 1">
+        <div v-for="allCats in [Categorisation.getAll()]">
+            <div v-for="(key, index) in allCats.keys()" :key="key.name">
+                <h2>{{ key.name }}</h2>
+                <p v-if="key.description">
+                    {{ key.description }} <a v-if="key.learnMore" :href="key.learnMore">Learn more...</a>
```
Comment (MattIPv4) [inline src/kubernetes-tool/templates/categorisation_view.vue]: Use the do-vue ExternalLink component for the learn more link, as they're external.

File: src/kubernetes-tool/templates/app.vue (hunk 3 above)
Comment (JakeMakesStuff) [reply src/kubernetes-tool/templates/app.vue]: Yeah, I forgot to mention this bug. I feel like it should wait until any designs from @AmNotADev (although I'd like your opinion on this)

Comment (MattIPv4) [reply src/kubernetes-tool/templates/app.vue]: The plan was to get this merged and then for Raven to do their UI review whenever they have time and create issues on the repo for changes. As such, do you want to just make a minimum fix for this and we can revise the design later?

Comment (JakeMakesStuff) [reply src/kubernetes-tool/templates/app.vue]: Ok, shall do!

**Comment by MattIPv4** [comment]
Currently blocked by https://github.com/do-community/do-vue/pull/5 & DOCOM#1510

**Pull request merged**